### PR TITLE
clarification on the usage of term "REST"

### PR DESCRIPTION
**Pull request by tfrauenstein**

### CODE REVIEW

**Comment by fmueller** [comment]
Can we place this footnote more prominent? It's quite hidden and can be overseen.

**Comment by hjacobs** [comment]
Can we ask @stilkov to review? :smirk:

**Comment by stilkov** [comment]
FWIW, I have no objections to the wording of the footnote, and FTR, I still object to the reasoning in the “Hypermedia” part of your guidelines, and believe you’re missing out.

**Comment by tfrauenstein** [comment]
@fmueller I think the footnote is positioned sufficiently prominent -- it is final statement of the principles section immediately following the introduction.

File: design-principles/DesignPrinciples.md [hunk 1]
```diff
@@ -59,3 +59,8 @@ general RESTful API design style:
 * InfoQ eBook: [Web APIs: From Start to Finish](http://www.infoq.com/minibooks/emag-web-api)
 * Lessons-learned blog: [Thoughts on RESTful API Design](http://restful-api-design.readthedocs.org/en/latest/)
 * Fielding Dissertation: [Architectural Styles and the Design of Network-Based Software Architectures](http://www.ics.uci.edu/~fielding/pubs/dissertation/top.htm)
+
+*Footnote:* 
+Per definition of R.Fielding REST APIs have to support HATEOAS (maturity level 3). Our guidelines do not strongly advocate for full REST compliance, but limited hypermedia usage, e.g. for pagination (see [Hypermedia Section](../hyper-media/Hypermedia.md#Hypermedia) below). However, we still use the term "RESTful API", due to the absence of an alternative established term and to keep it like the very majority of web service industry that also use the term for their REST approximations — in fact, in today's industry full HATEOAS compliant APIs are a very rare exception.
```
Comment (tkrop) [inline design-principles/DesignPrinciples.md]: Minor: R.Fielding -> R. Fielding

**Comment by tkrop** [comment]
While this location is a good choice because of the context, I although agree that it could be placed more prominent in the Introduction section. Anyhow, as long as we do not provoke a shit storm by further using the term *RESTful* we should go forward with this solution.

**Comment by hjacobs** [comment]
:+1:

**Comment by tkrop** [comment]
👍

**Pull request merged**